### ISSUE TITLE
Copy-Paste error in InputSystemUIInputModule

### ISSUE THREAD

**Issue by cgbeutler** [Unity-Technologies/InputSystem#831]
The below checks if `trackedDeviceOrientationAction` is null but it should check if `trackedDevicePositionAction` is null
https://github.com/Unity-Technologies/InputSystem/blob/a37a3801e190d1eb2c871ee221ea08415cedfdd4/Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs#L917-L919

**Comment by cgbeutler** [comment]
I would recommend introducing a simple helper function to reduce code and copy-paste errors. Something like:

```
        private void _hookActionIfNotNull( InputAction ia )
        {
            if ( ia == null ) return;

            ia.performed += m_OnActionDelegate;
            ia.canceled += m_OnActionDelegate;
        }

        private void HookActions()
        {
            if ( m_ActionsHooked )
                return;

            m_ActionsHooked = true;
            if ( m_OnActionDelegate == null )
                m_OnActionDelegate = OnAction;

            _hookActionIfNotNull( m_PointAction?.action );
            _hookActionIfNotNull( m_MoveAction?.action );
            _hookActionIfNotNull( m_LeftClickAction?.action );
            _hookActionIfNotNull( m_RightClickAction?.action );
            _hookActionIfNotNull( m_MiddleClickAction?.action );
            _hookActionIfNotNull( m_SubmitAction?.action );
            _hookActionIfNotNull( m_CancelAction?.action );
            _hookActionIfNotNull( m_ScrollWheelAction?.action );
            _hookActionIfNotNull( m_TrackedDeviceOrientationAction?.action );
            _hookActionIfNotNull( m_TrackedDevicePositionAction?.action );
            _hookActionIfNotNull( m_TrackedDeviceSelectAction?.action );
        }
```

**Comment by cgbeutler** [comment]
Looks like the same issue exists in the unhook action function

**Comment by Rene-Damm** [comment]
Thank you for reporting this. Should be fixed in #906.

**Comment by cgbeutler** [comment]
No prob. @Rene-Damm you should check out #830, too. I think that one would be high priority. I was gonna use action reference till I realized it was broken.
